tests/k8st/deploy_resources_on_kind_cluster.sh

```diff
@@ -104,6 +104,17 @@ done
 echo "client and webserver pods are running."
 echo
 
+echo "Deploy Calico apiserver"
+${kubectl} create -f https://docs.projectcalico.org/master/manifests/apiserver.yaml
+openssl req -x509 -nodes -newkey rsa:4096 -keyout apiserver.key -out apiserver.crt -days 365 -subj "/" -addext "subjectAltName = DNS:calico-api.calico-apiserver.svc"
+${kubectl} create secret -n calico-apiserver generic calico-apiserver-certs --from-file=apiserver.key --from-file=apiserver.crt
```

Comment:
Does the following `patch apiservice` need the apiserver to be ready?  I'm a little surprised that there is no wait here.

Reply:
Yes, you are right. I should move `wait` above after patch is done.